### PR TITLE
Update dataset card of wino_bias

### PR DESCRIPTION
**Pull request by JieyuZhao**
Updated the info for the wino_bias dataset.

### CODE REVIEW

File: datasets/wino_bias/README.md [hunk 1]
```diff
@@ -81,18 +79,19 @@ English
...
+    - word_sense = This is the word sense of the word in Column tokens. [set to '-' currently]
+    - speaker/author = This is the speaker or author name where available. [only "Speaker#1" currently]
+    - ner_tags = These columns identifies the spans representing various named entities. For documents which do not have named entity annotation, each line is represented with an "*". [set all to '*' now]
+    - verbal_predicates = There is one column each of predicate argument structure information for the predicate mentioned in predicate_lemma. If there are no predicates tagged in a sentence this is a single column with all rows marked with an "*". 
+    - coreference = Coreference chain information encoded in a parenthesis structure.
 
 ### Data Splits
 
-Single Split available
+- dev and test splits. For each split, there will be "pro_stereotype" and "anti_stereotype" splits. Each "pro_" and "anti_" splits will have "type1" and "type2" splits.
```
Comment (lhoestq) [inline datasets/wino_bias/README.md:94]: In the dataset script, only a train split is generated.
Where do these other splits come from ? Are they missing in the wino_bias dataset builder ?

File: datasets/wino_bias/README.md [hunk 2]
```diff
@@ -81,18 +79,19 @@ English
...
+    - predicate_lemma = The predicate lemma is mentioned for the rows for which we have semantic role information or word sense information. All other rows are marked with a "-". [set to '-' currently]
+    - predicate_framenet_id = This is the PropBank frameset ID of the predicate in predicate_lemma. [set to '-' currently]
+    - word_sense = This is the word sense of the word in Column tokens. [set to '-' currently]
+    - speaker/author = This is the speaker or author name where available. [only "Speaker#1" currently]
+    - ner_tags = These columns identifies the spans representing various named entities. For documents which do not have named entity annotation, each line is represented with an "*". [set all to '*' now]
+    - verbal_predicates = There is one column each of predicate argument structure information for the predicate mentioned in predicate_lemma. If there are no predicates tagged in a sentence this is a single column with all rows marked with an "*". 
+    - coreference = Coreference chain information encoded in a parenthesis structure.
```
Comment (lhoestq) [inline datasets/wino_bias/README.md:90]: There's no coreference field defined in the dataset builder

File: datasets/wino_bias/README.md [hunk 3]
```diff
@@ -81,18 +79,19 @@ English
...
+    - parse_bit = This is the bracketed structure broken before the first open parenthesis in the parse, and the word/part-of-speech leaf replaced with a *. The full parse can be created by substituting the asterix with the "([pos] [word])" string (or leaf) and concatenating the items in the rows of that column. When the parse information is missing, the first word of a sentence is tagged as "(TOP*" and the last word is tagged as "*)" and all intermediate words are tagged with a "*". [set to '-' currently]
+    - predicate_lemma = The predicate lemma is mentioned for the rows for which we have semantic role information or word sense information. All other rows are marked with a "-". [set to '-' currently]
+    - predicate_framenet_id = This is the PropBank frameset ID of the predicate in predicate_lemma. [set to '-' currently]
+    - word_sense = This is the word sense of the word in Column tokens. [set to '-' currently]
+    - speaker/author = This is the speaker or author name where available. [only "Speaker#1" currently]
```
Comment (lhoestq) [inline datasets/wino_bias/README.md:87]: the field name in the dataset builder is speaker

File: datasets/wino_bias/README.md [hunk 4]
```diff
@@ -10,15 +10,13 @@ licenses:
 multilinguality:
 - monolingual
 size_categories:
-- 10K<n<100K
+- 1K<n<10K
```
Comment (lhoestq) [inline datasets/wino_bias/README.md:13]:
```suggestion
- 10K<n<100K
```
There are 150335 examples in the dataset

**Review by lhoestq — COMMENTED** [review]
Thanks for adding the link and suggesting improvements to the dataset card.
Some of the information you added don't match what's defined in the wino_bias.py dataset builder unfortunately. Is the dataset builder missing some parts ?

**Comment by albertvillanova** [comment]
Thanks @JieyuZhao.

I think this PR was superseded by your other PRs:
- #1930
- #2152 

I'm closing this.

**Pull request closed without merging**